content/en/docs/concepts/services-networking/service.md

```diff
@@ -203,6 +203,8 @@ subsets:
 The name of the Endpoints object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
+{{< note >}} The name of the Endpoint must be the same as the name of the Service. {{< /note >}}
```

Comment:
- Kubernetes doesn't have an API type “Endpoint”; did you mean “[Endpoints](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/endpoints-v1/)”
- how does this requirement apply to the [EndpointSlice](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1/) API?


How about:
```suggestion
When you create an [Endpoints](docs/reference/kubernetes-api/service-resources/endpoints-v1/)
object for a Service, you set the name of the new Endpoints to be the same as the
name of the Service.
```

Reply:
You could also put a comment into the example Endpoints manifest @Shabirmean

Reply:
Thank you for the review @sftim. I've adopted the changes. Also I think EndpointSlices don't have this requirement. The association is done via adding an annotation.